sdks/python/apache_beam/runners/worker/data_plane.py

```diff
@@ -243,7 +274,7 @@ def output_stream(
       instruction_id,  # type: str
       transform_id  # type: str
   ):
-    # type: (...) -> ClosableOutputStream
+    # type: (...) -> SizeBasedBufferingClosableOutputStream
```

Comment:
`ClosableOutputStream` seems to not actually be used anywhere any more, because `ClosableOutputStream.create()` returns `SizeBasedBufferingClosableOutputStream`.  `ClosableOutputStream` does not implement the necessary `maybe_flush` method that's required everywhere we were referring to `ClosableOutputStream`

Reply:
We're also thinking about adding a time-based one. 

Let's add a no-op maybe_flush method to the baseclass and keep ClosableOutputStream everywhere.

Reply:
fixed.